SU-HKKU/cardinal_service_profile
# CSD-465 Updates from September 9 QA session

# READY FOR REVIEW

# Summary
- Updates from September 9 QA session

# Needed By (Date)
- 09/11

# Urgency
- medium

# Steps to Test

1. Pull this branch.
2. Clear the cache.
3. Check for the following:
- The spotlight view image is now 200x200
- Quote-only font-weight:400 and typography(2)
- The related resources view should not have additional margin
- Added see all resources button
- Added spacing for opportunity, resources, and stories button
- Updated font-weight for resources title
- Added border-top to opportunity dimension
- Added spacing for information card on opportunity sidebar

# Associated Issues and/or People
- [CSD-465](https://stanfordits.atlassian.net/browse/CSD-465)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
